### ISSUE TITLE
Edit Profile page

### ISSUE THREAD

**Issue by aayushkothari11** [djunicode/DJ-Comps-Student-Info-Portal#69]
So basically at the time of demo to add new object for example research paper the form only had 4-5 fields but the database schema has more fields which could be added.
It has to be checked for all the models.

**Comment by aayushkothari11** [comment]
Okie after @siddhant-soni added images to all the forms. The only thing left is ManyToMany Fields like BE project teammates and stuff like that. 
Do we really require that @shahsaumya or should we just leave that for now?

**Comment by shahsaumya** [comment]
No, it's fine. Do not complicate the models now. Each student can put up their BE project details khudse in edit profile. 

Finish other stuff then we'll see if this is required or not

**Comment by shahsaumya** [comment]
If everything else is done, you can close this issue.
